### PR TITLE
Bulk emails in docker

### PR DESCRIPTION
**Pull request by tdelatorre**

### CODE REVIEW

File: docker/config.py [hunk 1]
```diff
@@ -45,13 +45,17 @@
...
 
+
 #########################################
 ## EMAIL
 #########################################
 # https://docs.djangoproject.com/en/3.1/topics/email/
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 DEFAULT_FROM_EMAIL = os.getenv('DEFAULT_FROM_EMAIL', 'system@taiga.io')
 ENABLE_EMAIL = os.getenv('ENABLE_EMAIL', False) in (True, 'True')
+CHANGE_NOTIFICATIONS_MIN_INTERVAL = 30  # seconds
```
Comment (yamila-moreno) [inline docker/config.py:56]: Better 60 seconds

File: taiga/projects/notifications/services.py [hunk 2]
```diff
@@ -563,3 +567,16 @@ def make_ms_thread_index(msg_id, dt):
...
+    with advisory_lock("send-notifications-command", wait=False) as acquired:
+            if acquired:
+                qs = HistoryChangeNotification.objects.all().order_by("-id")
+                for change_notification in iter_queryset(qs, itersize=100):
+                    try:
+                        send_sync_notifications(change_notification.pk)
+                    except HistoryChangeNotification.DoesNotExist:
+                        pass
+            else:
+                print("Other process already running")
```
Comment (bameda) [inline taiga/projects/notifications/services.py:582]: This else statement should be removed, it has no sense in a service

File: docker/config.py (hunk 1 above)
Comment (tdelatorre) [reply docker/config.py]: Done!

File: taiga/projects/notifications/services.py (hunk 2 above)
Comment (tdelatorre) [reply taiga/projects/notifications/services.py]: Done!

**Pull request merged**